src/op_mode/restart_dhcp_relay.py

```diff
@@ -39,6 +40,9 @@
         if not c.exists_effective('service dhcp-relay'):
             print("DHCP relay service not configured")
         else:
+            if commit_in_progress():
+                print('Can not restart until commit in progress')
```

Comment:
`Can not restart DHCP relay until commit in progress as a commit is still in progress`...